apps/ui/src/networks/offchain/index.ts

```diff
@@ -63,7 +63,7 @@ export function createOffchainNetwork(networkId: NetworkID): Network {
   return {
     readOnly: true,
     name: networkId === 's-tn' ? 'Snapshot (testnet)' : 'Snapshot',
-    avatar: '',
+    avatar: 'ipfs://bafkreifsxa2m5xzqx5jiq64oswbyyb6qmi6faykdjaipv2vsggkqf55gzm',
```

Comment:
There is too less space around logo, but anyway we don't want to show Snapshot logo as a network for space preview